<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-A97EKAP" name="FailTest" tests="1" failures="1" timestamp="19 May 2020 15:04:09 GMT" time="65.339" errors="0">
  <testcase name="failvalidateTitle" time="0.326" classname="Academy.FailCase">
    <failure type="java.lang.AssertionError" message="expected [Feature fail title] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: expected [Feature fail title] but found [FEATURED COURSES]
	at Academy.FailCase.failvalidateTitle(FailCase.java:31)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- failvalidateTitle -->
</testsuite> <!-- FailTest -->
